haystack/components/eval/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .sas_evaluator import SASEvaluator
```

Comment:
This, and the other core evaluators, should go into `haystack.components.evaluators` instead.